src/intentLogic/yes.js

```diff
@@ -0,0 +1,23 @@
+const get = require("simple-get-promise").get;
+const asJson = require("simple-get-promise").asJson;
+
+const helpers = require('../helpers');
+const speech = require('../speech').speech;
+const sound = require('../speech').sound;
+
+const config = require("../../tmp/config.json");
+const CAPI_API_KEY = config.capi_key;
+
+module.exports = function () {
+	if(this.event.session.attributes.lastIntent){
```

Comment:
You have to call this.event.session.attributes allot. Maybe you can alias to a var somehow and expose it in a module to save your fingers.